Dockerfile

```diff
@@ -2,18 +2,31 @@
 # Use go-toolset as the builder image
 # Once built, copys GO executable to a smaller image and runs it from there
 # FROM registry.redhat.io/ubi8/go-toolset as builder
-FROM quay.io/projectquay/golang:1.20 as builder
+# FROM quay.io/projectquay/golang:1.20 as builder
+
+# WORKDIR /go/src/app
+
+# COPY go.mod go.sum ./
+
+# RUN go mod download
+
+# COPY . .
+
+# USER 0
+
+# RUN make
+
+FROM registry.access.redhat.com/ubi8/go-toolset:1.20.10 AS builder
```

Comment:
Curios, does this AS cap or non cap matter?

Reply:
I think it should be okay? That image was able to be built when I tested it locally, but I can also change it back to lower case to match the style!

Reply:
Thanks 👍